src/cls/fifo/cls_fifo.cc

```diff
@@ -33,6 +33,8 @@ static constexpr auto CLS_FIFO_MAX_PART_HEADER_SIZE = 512;
 
 static std::uint32_t part_entry_overhead;
 
+#define __le64 ceph_le64
```

Comment:
i'd suggest just s/__le64/ceph_le64/ and s/__le32/ceph_le32/ in this file. the reason why we use `__le64` and `__le32` in `include/cephfs.h` and `rados.h` is to share this definition with linux kernel, see 
https://github.com/ceph/ceph/blob/240c46a75a44cb9363cf994cb264e9d7048c98a1/src/include/ceph_fs.h#L19-L30

and 

https://github.com/torvalds/linux/blob/master/include/uapi/linux/types.h#L29-L34 . where `__le64` and `__le32` are defined as endian-dependent types.

but this file is only used in user-space application, and there is no need to be compiled along with by kernel, so i think `ceph_le64` and `ceph_le32` are right types for these variables if we want to store them in little-endian even on big-endian machines.

Reply:
@tchaikov 
Thanx for the info...

Will replace these instances